cli/lib/kontena/cli/stacks/install_command.rb

```diff
@@ -51,21 +57,33 @@ def install_dependencies
           cmd.concat ['-v', "#{key}=#{value}"]
         end
 
+        cmd << '--dry-run' if dry_run?
+
         cmd << '--no-deploy' unless deploy?
 
         cmd << dependency['stack']
-        Kontena.run!(cmd)
+        dependency_result = Kontena.run!(cmd)
+        dependency_result_variables = dependency_result['variables'] ||= {}
```

Comment:
The `foo_bar = foo['bar'] ||= {}` is a little surprising, not sure how the `||=` assignment actually behaves there.

Reply:
It's a typo but it does work, it sets both variables unless the middle one is already defined.

```
> foo = bar ||= 1
> foo 
=> 1
> bar 
=> 1
```